.circleci/config.yml

```diff
@@ -0,0 +1,234 @@
+version: 2
+
+jobs:
+  test-3.6: &test-template
+  # Job to run tests test on python 3.6
+    docker:
+      - image: circleci/python:3.6
+
+    environment:
+      PYTHON_BIN: python3
+      PYTHON_VERSION: 3.6
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v2-test-dependencies-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}
```

Comment:
I know I didn't have it. But I received an email recently that they adviced to use `{{ arch }}` as an extra key in case we have binary cached objects. I think that the majority of dependencies are pure python, but to be on the safe side it is better to put it.